src/components/atoms/CardHeader/CardHeader.tsx

```diff
@@ -0,0 +1,41 @@
+import { styled } from '@mui/material/styles';
+import { ReactNode } from 'react';
+import { Box, Typography } from 'src/components';
+
+export type CardHeaderProps = {
+  title: string;
+  description: string;
+  actions?: ReactNode;
+  children?: ReactNode;
+};
+
+const HeaderStyled = styled(Box)(({ theme: { palette } }) => ({
+  borderRadius: '4px 4px 0px 0px',
+  backgroundColor: palette.uiWhite[500],
+  display: 'flex',
+  alignItems: 'flex-start',
+  justifyContent: 'space-between',
+  padding: '16px 24px',
+  borderBottom: `1px solid ${palette.uiCoolGray[200]}`,
+  minHeight: '48px',
+}));
+
+export const CardHeader = ({
+  title,
```

Comment:
I would also always leave space for the `sx` prop in case the dev needs to override or add extra styling, they can do it

Reply:
+1